server/server.js

```diff
@@ -22,6 +23,37 @@ app.use(express.static(reactStaticDir));
 app.use(express.static(uploadsStaticDir));
 app.use(express.json());
 
+// creating a new user
+app.post('/api/users', async (req, res, next) => {
+  const { username, password } = req.body;
+  try {
+    const query = 'INSERT INTO "Users" (username, password) VALUES ($1, $2)';
+    await db.query(query, [username, password]);
+    res.status(201).json({ message: 'User created successfully' });
+  } catch (error) {
+    next(error);
```

Comment:
What code is handling this error? I don't see any error middleware being installed.